src/main/java/org/isf/patient/gui/PatientInsertExtended.java

```diff
@@ -2160,11 +2183,51 @@ private JPanel getJRightPanel() {
 				jRightPanel.add(photoPanel, BorderLayout.NORTH);
 			}
 			jRightPanel.add(getJNoteScrollPane(), BorderLayout.CENTER);
+			jRightPanel.add(getConsensus(), BorderLayout.SOUTH);
 
 		}
 		return jRightPanel;
 	}
 
+	private JPanel getConsensus() {
+		try {
+			if (patient != null && patient.getCode() != null) {
+				consensus = this.patientConsensusManager.getPatientConsensusByUserId(patient.getCode()).get();
+			} else {
+				consensus = new PatientConsensus();
+			}
+		} catch (OHServiceException e) {
+			consensus = new PatientConsensus();
+			LOGGER.debug(e.getMessage());
+		}
+		JPanel panel = new JPanel();
+		panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));
+
+		JCheckBox checkbox = new JCheckBox(MessageBundle.getMessage("angal.patient.consensus.consensus.txt"));
+		checkbox.addActionListener(e -> consensus.setConsensusFlag(!consensus.isConsensusFlag()));
+		checkbox.setSelected(consensus.isConsensusFlag());
+		panel.add(checkbox);
+
+		checkbox = new JCheckBox(MessageBundle.getMessage("angal.patient.consensus.service.txt"));
+		checkbox.addActionListener(e -> consensus.setServiceFlag(!consensus.isServiceFlag()));
+		checkbox.setSelected(consensus.isServiceFlag());
+		panel.add(checkbox);
+
+		checkbox = new JCheckBox(MessageBundle.getMessage("angal.patient.consensus.administrative.txt"));
+		checkbox.addActionListener(e -> consensus.setAdministrativeFlag(!consensus.isAdministrativeFlag()));
+		checkbox.setSelected(consensus.isAdministrativeFlag());
+		panel.add(checkbox);
+
+		panel.setBorder(BorderFactory.createCompoundBorder(
+						BorderFactory.createCompoundBorder(
```

Comment:
is this correct? `createCompoundBorder()` within another `createCompoundBorder()`; doesn't this look odd?